rust/cubesql/cubesql/src/compile/engine/udf.rs

```diff
@@ -1350,3 +1350,71 @@ pub fn create_generate_series_udtf(with_catalog_prefix: bool) -> TableUDF {
         &fun,
     )
 }
+
+pub fn create_pg_expandarray_udtf() -> TableUDF {
+    let fields = vec![
+        Field::new("x", DataType::Int64, true),
+        Field::new("n", DataType::Int64, false),
+    ];
+    let fields_cloned = fields.clone();
```

Comment:
Is there a better way to do this?

The fields are cloned due to the fact they are moved both in fun and return_type, and another clone is required in the functions themselves as Field does not implement the Copy trait.